test/parallel/test-timers-unenroll-unref-interval.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const common = require('../common');
+const timers = require('timers');
+
+const keepOpen = setTimeout(() => {
+  common.fail('Test timed out.');
+}, common.platformTimeout(500)); // Keep event loop open.
```

Comment:
> I don't think that's the correct interpretation.

Would it help if I set up a meeting with you to explain what actually happens with timers and why I am asserting that would be broken behavior?

> Clarification: Yes, the unref'ed intervals will fire before the timer. However, the test will still fail if the immediates wrapped inside the unref'ed intervals don't fire before the timer, and that's what's not guaranteed.

Thanks, I'll fix this so that it more precisely relies on the event loop specifics.